### PR TITLE
Polls 2023-05-12

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding GNO (Gnosis) to Spark Protocol.
+
+### Spark Onboarding Parameters
+
+**Token:** GNO  
+**Token Address:** 0x6810e776880C02933D47DB1b9fc05908e5386b96
```
Comment (astronautthis) [inline governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:34]:
```suggestion
**Token Address:** [0x6810e776880C02933D47DB1b9fc05908e5386b96](https://etherscan.io/address/0x6810e776880C02933D47DB1b9fc05908e5386b96)
```

File: governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+### Spark Onboarding Parameters
+
+**Token:** GNO  
+**Token Address:** 0x6810e776880C02933D47DB1b9fc05908e5386b96  
+**Oracle:** Maker Medianizer via adaptor  
+**Oracle Provider:** Chronicle  
+**Borrow Enabled:** No  
+**Collateral Enabled:** Yes  
+**[Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled:** Yes  
+**Isolation Mode Debt Ceiling:** 5 million USD
```
Comment (astronautthis) [inline governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:40]: Should this be DAI or USD?

File: governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+**Token:** GNO  
+**Token Address:** 0x6810e776880C02933D47DB1b9fc05908e5386b96  
+**Oracle:** Maker Medianizer via adaptor  
+**Oracle Provider:** Chronicle  
+**Borrow Enabled:** No  
+**Collateral Enabled:** Yes  
+**[Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled:** Yes  
+**Isolation Mode Debt Ceiling:** 5 million USD  
+**Loan To Value:** 20%
```
Comment (astronautthis) [inline governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:41]: Does it make sense to link Aave documentation here e.g. https://docs.aave.com/risk/v/master/asset-risk/risk-parameters#loan-to-value? Same for those below

File: governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+* It is expected that this executive vote will take place within 30 days of this poll passing, absent external factors.
+* If the executive vote passes, then these changes will become active in Spark Protocol and the Maker Protocol after the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
+
+---
+
+## Resources
+
+Further information about Spark Protocol can be found at its [Documentation Hub](https://docs.sparkprotocol.io/hub/).
```
Comment (astronautthis) [inline governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:70]: Potentially also Aave documentation if you think it makes sense - https://docs.aave.com

File: governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+---
+title: Onboard GNO to Spark Protocol - May 15, 2023
+summary: Onboard GNO (Gnosis) to Spark Protocol with included parameters. Reduce Maker GNO-A debt ceiling to zero.
```
Comment (astronautthis) [inline governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:3]:
```suggestion
summary: Signal your support or opposition to onboarding GNO (Gnosis) to Spark Protocol with the following parameters.
```

Comment (astronautthis) [reply governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:3]: Happy to also leave it as before if the DC change should feature in the summary.

Comment (LongForWisdom) [reply governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:3]: Yep, change here was intentional.

File: governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/polls/Onboard GNO to Spark Protocol - May 15, 2023.md:40]: USD is correct. I checked with Sam.

**Pull request merged**